player/lua/select.lua

```diff
@@ -636,8 +636,13 @@ mp.add_key_binding(nil, "menu", function ()
     input.select({
         prompt = "",
         items = labels,
+        keep_open = true,
         submit = function (i)
             mp.command(commands[i])
+
+            if not commands[i]:find("^script%-binding select/") then
```

Comment:
What if next select does not open? For example when there is no history or something like that?

Reply:
`submit` doesn't fail as long as console doesn't downright crash. The menu just stays open and you can keep using it in this case. It may even be better since I'd like to add "Edit config file" and "Edit key bindings" to it and a user could see to enable `--save-watch-history` and immediately edit the config file (of course the whole reason history is still shown even without `--save-watch-history` is to make it discoverable to who would like to enable it).

Reply:
Ok, we can have it. We can always fix if it causes issues.